framework/wazuh/core/cluster/utils.py

```diff
@@ -25,7 +25,7 @@
 execq_lockfile = join(common.wazuh_path, "var/run/.api_execq_lock")
 
 
-def read_cluster_config(config_file=common.ossec_conf) -> typing.Dict:
+def read_cluster_config(config_file=common.ossec_conf, run_import=False) -> typing.Dict:
```

Comment:
```suggestion
def read_cluster_config(config_file=common.ossec_conf, from_import=False) -> typing.Dict:
```
I believe this name is more accurate. Take this as a suggestion, not a requirement.